tests/asyncio/test_asyncio_server.py

```diff
@@ -115,6 +115,7 @@ def test_async_modes_wsgi(self):
                           async_mode='gevent_uwsgi')
         self.assertRaises(ValueError, asyncio_server.AsyncServer,
                           async_mode='threading')
+        del sys.modules['engineio.async_drivers.gevent']
```

Comment:
This should be guarded so tests work on native environments when gevent is not available:
```python
        if 'engineio.async_drivers.gevent' in sys.modules
            del sys.modules['engineio.async_drivers.gevent']
```